lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -101,6 +146,7 @@ class AwsCompileScheduledEvents {
             let Description;
             let method;
             let roleArn;
+            let ScheduleExpressionTimezone;
```

Comment:
Snake case for new vars; this file will eventually get cleaned up.

Reply:
Will make the change, was going the way the guys did before.